File: Examples/CSharp/ProgrammersGuide/Working-with-Masters/CheckMasterPresencebyName.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Diagram.Examples.CSharp.ProgrammersGuide.Working_with_Masters
{
    public class CheckMasterPresencebyName
    {
        public static void Run() 
        {
            // ExStart:CheckMasterPresencebyName
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Master();

            // Call the diagram constructor to load diagram from a VDX file
            Diagram diagram = new Diagram(dataDir + "Basic Shapes.vss");

            // Set master name
            string masterName = "VNXe3100 Storage Processor Rear";
            // check master object by name
            bool isPresent = diagram.Masters.IsExist(masterName);

            Console.WriteLine("Master Presence : " + isPresent);
            // ExEnd:CheckMasterPresencebyName
        }
    }
}

File: Examples/VisualBasic/Working-with-Masters/CheckMasterPresencebyName.vb

Imports Aspose.Diagram
Imports System

Public Class CheckMasterPresencebyName
    Public Shared Sub Run()
        ' ExStart:CheckMasterPresencebyName
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_Master()

        ' Call the diagram constructor to load diagram from a VDX file
        Dim diagram As New Diagram(dataDir & Convert.ToString("Basic Shapes.vss"))

        ' Set master name
        Dim masterName As String = "VNXe3100 Storage Processor Rear"
        ' Check master object by name
        Dim isPresent As Boolean = diagram.Masters.IsExist(masterName)

        Console.WriteLine("Master Presence : " & isPresent)
        ' ExEnd:CheckMasterPresencebyName
    End Sub
End Class
